Goal: Document how password strength is checked
Fixes: #115380

Status wrt upstream: Proposed but not applied yet

Index: shadow-4.0.13/man/passwd.1.xml
===================================================================
--- shadow-4.0.13.orig/man/passwd.1.xml	2005-10-05 07:35:27.325611228 +0200
+++ shadow-4.0.13/man/passwd.1.xml	2005-10-08 19:54:22.502118479 +0200
@@ -95,10 +95,16 @@
       </para>
 
       <para>
-        The user is then prompted for a replacement password. This password
-	is tested for complexity. As a general guideline, passwords should
-	consist of 6 to 8 characters including one or more from each of
-	following sets:
+        The user is then prompted twice for a replacement
+        password. The second entry is compared against the first and
+        both are required to match in order for the password to be
+        changed.
+      </para>
+
+      <para>
+        Then, the password is tested for complexity. As a general
+        guideline, passwords should consist of 6 to 8 characters including
+        one or more characters from each of the following sets:
       </para>
 
       <itemizedlist mark='bullet'>
@@ -119,11 +125,6 @@
 	is not suitably complex.
       </para>
 
-      <para>If the password is accepted, <command>passwd</command> will
-	prompt again and compare the second entry against the first. Both
-	entries are required to match in order for the password to be
-	changed.
-      </para>
     </refsect2>
 
     <refsect2 id='group_passwords'>
